src/components/data/data-utils.js

```diff
@@ -373,7 +383,7 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
   const [showInvalidEntryMethodWarning, setShowInvalidEntryMethodWarning] = useState(false);
   const [uploading, setUploading] = useState(false);
   const [deleteEmptyValues, setDeleteEmptyValues] = useState(false);
-  const [recordType, setRecordType] = useState(undefined);
+  const [recordType, setRecordType] = useState('');
```

Comment:
With this initialized to undefined, React warned about changing an uncontrolled component to be controlled when `setRecordType` was called with a string.